ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -11406,9 +11402,7 @@ static int zend_jit_switch(dasm_State **Dst, const zend_op *opline, const zend_o
 					|	cmp FCARG2a, jumptable->nNumUsed
 					|	jae >3
 					|.if X64
-					|	// TODO: DynASM stores .aword as 4-bytes even in 64-bit mode ???
-					|	movsxd r0, dword [FCARG2a * 4 + >4]
-					|	jmp r0
+					|	jmp aword [FCARG2a * 8 + >4]
```

Comment:
@dstogov While the jumptable itself seems to be generated correctly now, I have trouble generating the correct `jmp` here. The problem is that `>4` gets embedded as a 32-bit absolute offset. I tried doing something like

```
mov64 r0, >4
jmp aword [FCARG2a * 8 + r0]
```

but that isn't support by dynasm either.

Reply:
I'll check this tomorrow

Reply:
This and one different problem should be fixed at #5825